src/V3AstNodes.cpp

```diff
@@ -1073,6 +1073,33 @@ string AstBasicDType::prettyDTypeName() const {
 void AstNodeMath::dump(std::ostream& str) const { this->AstNode::dump(str); }
 void AstNodeUniop::dump(std::ostream& str) const { this->AstNodeMath::dump(str); }
 
+AstNodeMath* AstStdRandomize::newStdRandomize(FileLine* fl, AstNodeVarRef* varp, int offset,
+                                              AstMemberDType* memberp) {
+    AstStructDType* structDtp;
+    if (memberp)
+        structDtp = VN_CAST(memberp->subDTypep()->subDTypep(), StructDType);
+    else
+        structDtp = VN_CAST(varp->dtypep()->skipRefp(), StructDType);
+    if (structDtp) {
+        AstNodeMath* rand = nullptr;
+        auto* memberp = structDtp->membersp();
+        for (; memberp && memberp->nextp(); memberp = VN_CAST(memberp->nextp(), MemberDType))
+            ;
+        for (; memberp; memberp = VN_CAST(memberp->backp(), MemberDType)) {
```

Comment:
Comment why need to go to end first then backwards?  Would prefer not to if have choice.  Note there's a faster way to get there if have to which is headtailp, but that's private.  Also note backp also goes "up" not just backwards, so this might be confusing.  It'll still stop OK as "up" won't be a MemberDType.

Reply:
I thought I need to go backwards because I need to calculate member offsets (which I need for structs contained within structs, as their members' `lsb` is counted from the inner struct) but I just realized I could simply use the `lsb` from the `memberp` that points to the inner struct. So I'll get rid of that.